File: ruby-pr/53/ruby/code-quality-naming/item_manager.rb
class ItemManager
  # {fact rule=code-quality-naming@v1.0 defects=1}
  def initialize
    @stuff = nil
  end

  def do_thing(thing1, thing2)
    @stuff = {} if @stuff.nil?

    temp = thing1 + thing2
    @stuff[thing1] = temp
    
    @stuff.key?(thing1)
  end

  def get_thing(key)
    @stuff[key]
  end
  # {/fact}
end

File: php-pr/51/php/code-quality-naming/ItemManager.php
<?php

class ItemManager
{
    // {fact rule=code-quality-naming@v1.0 defects=1}
    private $stuff;

    public function doThing($thing1, $thing2)
    {
        if ($this->stuff === null) {
            $this->stuff = [];
        }

        $temp = $thing1 . $thing2;
        $this->stuff[$thing1] = $temp;
        
        return isset($this->stuff[$thing1]);
    }

    public function getThing($key)
    {
        return $this->stuff[$key] ?? null;
    }
    // {/fact}
}